site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -324,6 +324,32 @@ paths:
             403:
                $ref: '#/components/responses/NotAllowed'
 
+      get:
+         tags:
+            - devices
+         summary: Search devices for a tenant, with optional filters and paging options.
```

Comment:
IMHO it is also better to include `sorting` in the summary. How about `Search devices for a tenant with optional filters, paging and sorting options`?